scripts/ansible/install-ansible.sh

```diff
@@ -7,11 +7,30 @@ set -o errexit
 
 DIR=$(dirname "$0")
 
+function wait-apt-get {
+    i=0
+    echo "Checking for locks..."
+    # Check for locks
+    while fuser /var/lib/dpkg/lock > /dev/null 2>&1 ||
+          fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1 ||
+          fuser /var/lib/apt/lists/lock > /dev/null 2>&1; do
+        # Wait up to 600 seconds to lock to be released
+        if (( $i > 600 )); then
+            echo "Timeout waiting for lock."
+            exit 1
+        fi
+        echo "Waiting for apt/dpkg locks..."
+        ((i++))
+        sleep 1
+    done
+    apt-get ${1}
```

Comment:
This will take only the first argument, should we use `$@` instead?

Reply:
Absolutely, thanks for the good catch!